lib/integrations/awsoidc/listdatabases_test.go

```diff
@@ -68,15 +66,32 @@ func (m mockListDatabasesClient) DescribeDBInstances(ctx context.Context, params
 
 	if sliceEnd < totalInstances {
 		nextToken := fmt.Sprintf("%d", requestedPage+1)
-		ret.Marker = stringPointer(nextToken)
+		ret.Marker = aws.String(nextToken)
 	}
 
 	return ret, nil
 }
 
+func hasAuroraEngineFilter(filters []rdsTypes.Filter) bool {
+	for _, filter := range filters {
+		if *filter.Name == filterEngine {
```

Comment:
can we break early from the outer loop if we match a filter without aurora in it?

Reply:
Yes :+1: